src/Pages/Deworming.jsx

```diff
@@ -1,13 +1,20 @@
 import Navbar from "../components/Navbar";
 import Header from "../components/Header";
 import DewormingList from "../components/DewormingList";
+import { motion } from "framer-motion";
 
 function Deworming() {
   return (
     <>
-      <Header titleText={"Entwurmung"} />
-      <DewormingList />
-      <Navbar />
+        <Header titleText={"Entwurmung"} />
+      <motion.div
+        exit={{ opacity: 0 }}
+        animate={{ opacity: 1 }}
+        initial={{ opacity: 0 }}
+      >
+        <DewormingList />
+      </motion.div>
+        <Navbar />
```

Comment:
Please check the formatting, strings as attributes also don't need the curly braces around:
```suggestion
        <Header titleText="Entwurmung" />
```